server/views/partials/layout.njk

```diff
@@ -100,6 +100,11 @@
   <script src="/assets/govukFrontendInit.js"></script>
   <script src="/assets/moj/all.js"></script>
   <script src="/assets/print.js"></script>
+  <script src="/assets/applicationinsights-web.min.js"></script>
+  <script src="/assets/applicationinsights.js"></script>
+  <script nonce="{{ cspNonce }}">
+    window.applicationInsights.init('{{ applicationInsightsConnectionString }}', '{{ applicationInsightsRoleName }}', '{{ user.username }}');
```

Comment:
This line calls the `init` method of `applicationInsights` , passing in the connection string, application name (cloud role name), and the authenticated user name